x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/dashboard_sections/benchmarks_section.tsx

```diff
@@ -175,18 +187,28 @@ export const BenchmarksSection = ({
             grow={dashboardColumnsGrow.second}
             data-test-subj={DASHBOARD_TABLE_COLUMN_SCORE_TEST_ID}
           >
-            <CloudPostureScoreChart
-              compact
-              id={`${cluster.meta.assetIdentifierId}_score_chart`}
-              data={cluster.stats}
-              trend={cluster.trend}
-              onEvalCounterClick={(evaluation) =>
-                navToFindingsByClusterAndEvaluation(cluster, evaluation)
-              }
-            />
+            <ChartPanel
+              hasBorder={false}
+              panelStyles={css`
+                padding: 0 ${compactPadding};
+              `}
+            >
+              <CloudPostureScoreChart
+                compact
+                id={`${cluster.meta.assetIdentifierId}_score_chart`}
+                data={cluster.stats}
+                trend={cluster.trend}
+                onEvalCounterClick={(evaluation) =>
+                  navToFindingsByClusterAndEvaluation(cluster, evaluation)
+                }
+              />
+            </ChartPanel>
           </EuiFlexItem>
           <EuiFlexItem grow={dashboardColumnsGrow.third}>
             <RisksTable
+              containerStyles={css`
+                padding: 0 ${compactPadding};
+              `}
```

Comment:
I could move the `containerStyles` from `ChartPanel` to `CloudPostureScoreChart` and remove the ChartPanel. 
The padding needs to be applied to a container of the chart or risk table. Adding another div with padding cause misalignment issues where padding is being applied at the div level and not at the chart or table level. 
<img width="769" alt="image" src="https://user-images.githubusercontent.com/17135495/217605533-28b4ad63-aede-423b-be2d-6e960824c529.png">

We need to add padding around the correct flexGroup which contains the content.

Reply:
I'm dont think there should be a difference between passing stylings to the first wrapper inside a component and wrapping the same component with stylings. id prefer if we get on a call for that just so i can understand the case better.
Also, i think those components were already wrapped with those padding but it looks like they were removed by mistake i assume in this pr: https://github.com/elastic/kibana/pull/149566/files#diff-3d5d10f2f303033c8d9c525600f1cc08ca3ca962f9c1f390ffd971c09e2b2190

we didn't have misalignment problems between the summary and the benchmark section before and its not a part of the figma as well so i suspect its new. if it is, it would prefer if we just stick to what we had.

Also, it looks like in the figmas the benchmark section components are wider than the ones in the summary sections, it might be the reason those styles were removed, but i agree that it looks inconsistent and its better when the charts are the same size..

Reply:
jumped on a talk with @r4zr32d3k1l to understand exactly which way he wanted it to look, he prefers it without padding at all because the gap between the components is too big with the padding. we can remove those padding changes completely for now.